packages/itinerary-body/src/styled.js

```diff
@@ -267,7 +267,7 @@ export const IntermediateStops = styled.div`
 export const ItineraryBody = styled.div``;
 
 export const LegBody = styled.div`
-  color: #999;
+  color: #676767; /* from #999 to meet WCAG */
```

Comment:
We probably don't need a comment about the previous value here or any other place in this file.